cli/tests/integration/vendor_tests.rs

```diff
@@ -73,6 +73,31 @@ fn output_dir_exists() {
   assert!(status.success());
 }
 
+#[test]
+fn vendor_remote_module_test() {
+  let _server = http_server();
+  let t = TempDir::new();
+  let vendor_dir = t.path().join("vendor2");
+  t.write(
+    "my_app.ts",
+    "import { load } from 'https://x.nest.land/Yenv@1.0.0/mod.ts'; const env = await load({});",
```

Comment:
Really thanks for your attention .
Yes surely .
But one question I have is it will vendor data with which domain is stored in `specifier` , on issue which I mentioned the domain changed to strange domain it cause is `deno_graph` . Do think should I work on it to resolve it and save with the main url path ? 
and if you worked on `deno_graph` as I am seeing it's code right now , what is your command or reason for this changing on `url` ? I reached to one function which name is `resolve` and `ModuleGraph` and believe this cause this error .

Reply:
> Instead of an integration test, would you be able to add a test to tools/vendor/build.rs? This test is not ideal because it relies on a remote server and integration tests are slower to run than the tests in tools/vendor/build.rs.

We talk about redirection on issue, 
But as I said url change,  on tests on build.rs , how can I add `loader` with a remote url ?

Reply:
I don't think example on build.rs will work correct with this issue is we face a issue with path changing name by domain and host . For example if you are going to execute this command on this PR:
`deno vendor https://x.nest.land/Yenv@1.0.0/mod.ts --force
`
this will happen on `vendor/import_map.json` :  
```
{
  "imports": {
    "https://deno.land/": "./deno.land/",
    "https://krnvmnln2bt6577673oo4rze3ntyki3ob24k2qyxhsibzkqz6qaq.arweave.net/": "./krnvmnln2bt6577673oo4rze3ntyki3ob24k2qyxhsibzkqz6qaq.arweave.net/"
  }
}

```
if you look on graph module the domain is like a above .
As you know tests on build.rs generate file and can't exactly show this problem .
this commit bypass this error and keep working with any domain name .